cloudformation-template.json

```diff
@@ -37,7 +37,8 @@
 		"ApiGatewayRestApi": {
 			"Type": "AWS::ApiGateway::RestApi",
 			"Properties": {
-				"Name" : "Tachyon"
+				"Name" : "Tachyon",
+				"BinaryMediaTypes": [ "image~1*" ]
```

Comment:
I'm not sure what this syntax is, I thought it would be `image/*` or sometjing

Reply:
[Escaped slash](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-restapi.html#cfn-apigateway-restapi-binarymediatypes):
> Slashes must be escaped with `~1`. For example, `image/png` would be `image~1png` in the `BinaryMediaTypes` list.

Reply:
Cray cray